### ISSUE TITLE
Support node >= v4.1.2

### ISSUE THREAD

**Issue by AdamMagaluk** [zettajs/zetta#259]
Our tests are breaking with  v4.1.2 and above.

```
  1) Event Streams "after each" hook:
     Uncaught TypeError: invalid data
      at Socket.write (net.js:617:11)
      at Connection.write (/Users/ApigeeCorporation/Software/zetta/zetta/node_modules/spdy/lib/spdy/connection.js:515:36)
      at Scheduler.tickListener [as _tickListener] (/Users/ApigeeCorporation/Software/zetta/zetta/node_modules/spdy/lib/spdy/scheduler.js:67:23)
      at processImmediate [as _immediateCallback] (timers.js:374:17)
```

**Comment by wooliet** [comment]
Will there be backward compatibility with 0.12.x?

**Comment by AdamMagaluk** [comment]
@wooliet Yes we will continue to support 0.12.x with the zetta v1 release.

**Comment by AdamMagaluk** [comment]
Fixed in #269
